solidity/dashboard/src/components/coverage-pools/AddAmountToWithdrawalForm.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react"
+import { withFormik } from "formik"
+import FormInput from "../../components/FormInput"
+import { SubmitButton } from "../../components/Button"
+import Divider from "../../components/Divider"
+import MaxAmountAddon from "../MaxAmountAddon"
+import { normalizeAmount, formatAmount } from "../../forms/form.utils"
+import { KEEP } from "../../utils/token.utils"
+import TokenAmount from "../TokenAmount"
+import { useCustomOnSubmitFormik } from "../../hooks/useCustomOnSubmitFormik"
+import {
+  validateAmountInRange,
+  getErrorsObj,
+} from "../../forms/common-validators"
+import { lte } from "../../utils/arithmetics.utils"
+import useSetMaxAmountToken from "../../hooks/useSetMaxAmountToken"
+import {getSamePercentageValue} from "../../utils/general.utils";
+import {useSelector} from "react-redux";
+
+const AddAmountToWithdrawalForm = ({ tokenAmount, onSubmit, initialValue = "0", ...formikProps }) => {
+  const onSubmitBtn = useCustomOnSubmitFormik(onSubmit)
+  const onAddonClick = useSetMaxAmountToken("tokenAmount", tokenAmount)
+  const {
+    totalValueLocked,
+    covTotalSupply,
+  } = useSelector((state) => state.coveragePool)
```

Comment:
Can we pass these values via props?

Reply:
Fixed in c93eb5f8fc91da2d5482235944ede7e882cfc6ff